flutter/test/sentry_flutter_test.dart

```diff
@@ -95,9 +97,11 @@ void main() {
       final event =
           verify(transport.send(captureAny)).captured.first as SentryEvent;
 
-      expect(event.sdk!.integrations.length, 6);
+      expect(event.sdk!.integrations.length, 7);
```

Comment:
these tests counting the number of integrations are not great (my fault probably :P), lets consider refactoring it in the future to test the integration type, 1 by 1 or something that does not require a magic number here every time

Reply:
https://github.com/getsentry/sentry-dart/issues/390